From 3e8e8e8ef19c08cc16008f5732ce656b7ec658ad Mon Sep 17 00:00:00 2001
From: Peng Fan <Peng.Fan@freescale.com>
Date: Tue, 28 Apr 2015 11:08:05 +0800
Subject: [PATCH] MLK-10448-7 imx:mx6qpsabreauto add missed macro

Add CONFIG_DEFAULT_FDT_FILE macro and PHYS_SDRAM_SIZE.

Signed-off-by: Peng Fan <Peng.Fan@freescale.com>
---
 include/configs/mx6qsabreauto.h | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/include/configs/mx6qsabreauto.h b/include/configs/mx6qsabreauto.h
index d0981f2..b978821 100644
--- a/include/configs/mx6qsabreauto.h
+++ b/include/configs/mx6qsabreauto.h
@@ -13,7 +13,10 @@
 #define CONFIG_MXC_UART_BASE	UART4_BASE
 #define CONFIG_CONSOLE_DEV		"ttymxc3"
 #define CONFIG_MMCROOT			"/dev/mmcblk2p2"  /* SDHC3 */
-#if defined CONFIG_MX6Q
+#if defined CONFIG_MX6QP
+#define CONFIG_DEFAULT_FDT_FILE	"imx6qp-sabreauto.dtb"
+#define PHYS_SDRAM_SIZE		(2u * 1024 * 1024 * 1024)
+#elif defined CONFIG_MX6Q
 #define CONFIG_DEFAULT_FDT_FILE	"imx6q-sabreauto.dtb"
 #define PHYS_SDRAM_SIZE		(2u * 1024 * 1024 * 1024)
 #elif defined(CONFIG_MX6DL)
-- 
1.8.0

